scripts/jenkins/gitlint.ini

```diff
@@ -0,0 +1,20 @@
+[general]
+verbosity = 2
+
+[title-max-length]
+line-length=80
+
+[title-must-not-contain-word]
+words=wip,WIP
+
+[title-match-regex]
+regex=^SCRD-.*|.*\((bsc#[1-9]+\s?)+\)$
+
+[body-max-line-length]
+line-length=80
+
+[body-min-length]
+min-length=5
+
+[author-valid-email]
+regex = "[^@]+@suse.com"
```

Comment:
This doesn't look wide enough - you'll need to accommodate at least `@suse.de` too, and maybe others.